src/main/java/duke/Task.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+
+public abstract class Task {
+
+  protected String description;
+  protected boolean done;
```

Comment:
Do you think changing to isDone will better conform to our styleguide?
```suggestion
  protected boolean isDone;
```